telegram/ext/messagequeue.py

```diff
@@ -141,130 +181,189 @@ def stop(self, timeout: float = None) -> None:
-            process per time-window defined by :attr:`group_time_limit_ms`. Defaults to 20.
+            process per time-window defined by :attr:`group_time_limit_ms`. Defaults to
+            :attr:`telegram.constants.MAX_MESSAGES_PER_MINUTE_PER_GROUP`.
         group_time_limit_ms (:obj:`int`, optional): Defines width of *group-type* time-window used
             when each processing limit is calculated. Defaults to 60000 ms.
-        exc_route (:obj:`callable`, optional): A callable, accepting one positional argument; used
-            to route exceptions from processor threads to main thread; is called on ``Exception``
+        error_handler (:obj:`callable`, optional): A callable, accepting 1 positional argument.
+            Used to route exceptions from processor thread to main thread. Is called on `Exception`
             subclass exceptions. If not provided, exceptions are routed through dummy handler,
-            which re-raises them.
-        autostart (:obj:`bool`, optional): If :obj:`True`, processors are started immediately after
-            object's creation; if :obj:`False`, should be started manually by :attr:`start` method.
-            Defaults to :obj:`True`.
+            which re-raises them. If :attr:`dispatcher` is set, error handling will *always* be
+            done by the dispatcher.
+        exc_route (:obj:`callable`, optional): Deprecated alias of :attr:`error_handler`.
+        autostart (:obj:`bool`, optional): If :obj:`True`, both default delay queues are started
+            immediately after object's creation. Defaults to :obj:`True`.
 
     """
 
     def __init__(
         self,
-        all_burst_limit: int = 30,
+        all_burst_limit: int = constants.MAX_MESSAGES_PER_SECOND,
         all_time_limit_ms: int = 1000,
-        group_burst_limit: int = 20,
+        group_burst_limit: int = constants.MAX_MESSAGES_PER_MINUTE_PER_GROUP,
         group_time_limit_ms: int = 60000,
         exc_route: Callable[[Exception], None] = None,
         autostart: bool = True,
+        error_handler: Callable[[Exception], None] = None,
     ):
-        # create according delay queues, use composition
-        self._all_delayq = DelayQueue(
-            burst_limit=all_burst_limit,
-            time_limit_ms=all_time_limit_ms,
-            exc_route=exc_route,
-            autostart=autostart,
-        )
-        self._group_delayq = DelayQueue(
+        self.running = autostart
+        self.dispatcher: Optional['Dispatcher'] = None
+
+        if exc_route and error_handler:
+            raise ValueError('Only one of exc_route or error_handler can be passed.')
+        if exc_route:
+            warnings.warn(
+                'The exc_route argument is deprecated. Use error_handler instead.',
+                TelegramDeprecationWarning,
+                stacklevel=2,
+            )
+
+        self._delay_queues: Dict[str, DelayQueue] = {
```

Comment:
this dict should be accessible to make it easier to use existing delay queues as parents for new ones. Either just make `self._delay_queues` public, or even implement `__getitem__` to allow  for `mq['delay_queue_name']`